index.html

```diff
@@ -346,5 +346,51 @@ <h2 class="about-header">About<br />Myself</h2>
         </li>
       </ul>
     </section>
+    <footer class="footer">
+      <div class="layer"></div>
+      <div class="contact-me-first">
+        <h2 class="contact-me-header">Contact me</h2>
+        <p class="contact-me-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <div class="form-container">
+        <form action="https://formspree.io/f/#" method="post">
```

Comment:
- [ ] Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).
          - action value should be similar to https://formspree.io/f/{form_id} (Currently you are not using your form_id, please register on formspree and use your unique form id so that you'll recieve the message sent from your portfolio)

Reply:
I register that website and created a new form. After that I get this link "https://formspree.io/f/mdoyznbe". I inserted that link into my form tag. "mdoyznbe" is my form id.